analysers/analyser_osmosis_powerline.py

```diff
@@ -553,20 +635,23 @@ def __callback80_fix(self, res):
 
 ###########################################################################
 
+import time
 from .Analyser_Osmosis import TestAnalyserOsmosis
 
 class Test(TestAnalyserOsmosis):
     @classmethod
     def setup_class(cls):
         from modules import config
         TestAnalyserOsmosis.setup_class()
-        cls.analyser_conf = cls.load_osm("tests/osmosis_powerline_voltage.test.osm",
-                                         config.dir_tmp + "/tests/osmosis_powerline_voltage.test.xml")
+        cls.analyser_conf = cls.load_osm("tests/osmosis_powerline.test.osm",
+                                         config.dir_tmp + "/tests/osmosis_powerline.test.xml")
 
-    def test_class3(self):
+    def test_powergrid(self):
         with Analyser_Osmosis_Powerline(self.analyser_conf, self.logger) as a:
             a.analyser()
 
         self.root_err = self.load_errors()
-        self.check_err(cl="3", elems=[("node", "4")])
-        self.check_num_err(1)
+        self.check_err(cl="2", elems=[("node", "25874")])
```

Comment:
@frodrigo this test is failing but when running the tests, I have the following in the xml:
```xml
<error class="2">
<node id="25874" lat="46.19228681528" lon="4.47405870035" version="1" user="">
<tag k="power" v="tower" />
</node>
<way id="1909">
<tag k="power" v="line" />
<tag k="voltage" v="225000" />
</way>
<location lat="46.19228681528" lon="4.47405870035" />
</error>```

Am I mistaking something?

Reply:
it is because your analysis returns several osm objects (a node and its way). You need to specify both.

It will work with 
```
self.check_err(cl="2", elems=[("node", "25874"), ("way", "1909")])
```

Reply:
Good point :) It worked perfectly, thank you